### PR TITLE
committee members list style matches attendance

### PR DESCRIPTION
**Pull request by guushoekman**

### CODE REVIEW

File: pmg/templates/committee_detail.html [hunk 1]
```diff
@@ -165,54 +164,38 @@ <h3 class="no-results">No results.</h3>
...
+              <li class="media">
+                <div class="media-left">
+                  <a href="{{ item.member.pa_url }}">
+                    <img class="member-profile-pic media-object" src="{{ item.member.profile_pic_url or '/static/resources/images/no-profile-pic.svg' }}" alt="Profile picture: {{ item.member.name }}">
+                  </a>
+                </div>
+                <div class="media-body">
+                  <a href="{{ item.member.pa_url }}">{{ item.member.name }}</a>
+                  <div class="text-muted">{{ item.member.party.name }}</div>
+                  {% if item.chairperson == true %}
```
Comment (longhotsummer) [inline pmg/templates/committee_detail.html]: You can just use `{% if item.chairperson %}`

File: pmg/templates/committee_detail.html [hunk 2]
```diff
@@ -165,54 +164,38 @@ <h3 class="no-results">No results.</h3>
...
+                    <span class="label label-success">Chairperson</span>
+                  {% endif %}
+                </div>
+              </li>
+            </ul>
+          </div>
+        {% endfor %}
+      {% else %}
+      <div>
+        <span>We don't have membership info for this committee</span>
```
Comment (longhotsummer) [inline pmg/templates/committee_detail.html]: Please adjust this line in the *committee meeting* template to "committee meeting", if you haven't already?

File: pmg/templates/committee_detail.html [hunk 3]
```diff
@@ -139,7 +139,7 @@ <h3 class="no-results">No results.</h3>
       <table class="table table-striped">
         {% for item in recent_questions %}
           <tr class="item">
-            <td class="date"><nobr>{{ (item.start_date or item.date)|pretty_date }}</nobr></div>
+            <td class="date"><nobr>{{ (item.start_date or item.date)|pretty_date }}</nobr>
```
Comment (longhotsummer) [inline pmg/templates/committee_detail.html]: Should be a `</td>` at the end

**Comment by longhotsummer** [comment]
Just missing the closing TD, otherwise looks good.

File: pmg/templates/committee_detail.html (hunk 3 above)
Comment (guushoekman) [inline pmg/templates/committee_detail.html]: Ah alright. I noticed a random `</div>` so I removed it but didn't look properly

**Pull request merged**